code/datums/ai/objects/mod.dm

```diff
@@ -0,0 +1,48 @@
+/// An AI controller for the MODsuit pathfinder module. It's activated by implant and attaches itself to the user.
+/datum/ai_controller/mod
```

Comment:
suit fails to go through any doors (if no access, open the door and says connection lost)

Reply:
Yea I cant really improve this without substantial JPS reworking

Reply:
not happy but its good enough for this PR